src/simulator/src/modules/Splitter.js

```diff
@@ -183,53 +178,12 @@ export default class Splitter extends CircuitElement {
         this.prevInpValue = undefined
     }
 
-    /**
-     * @memberof Splitter
-     * fn to process verilog of the element
-     * @return {JSON}
-     */
-    processVerilog() {
```

Comment:
Why was this removed?

Reply:
Here too generateVerilog is defined twice for the same object

Reply:
Got it.